Example/Example/ViewController.swift

```diff
@@ -246,38 +246,20 @@ class ViewController: UIViewController {
             }
         }
     }
-
-    override func observeValue(forKeyPath keyPath: String?,
-                               of object: Any?,
-                               change: [NSKeyValueChangeKey: Any]?,
-                               context: UnsafeMutableRawPointer?)
-    {
-        // Only handle observations for the playerItemContext
-        guard context == &playerItemContext else {
-            super.observeValue(forKeyPath: keyPath,
-                               of: object,
-                               change: change,
-                               context: context)
-            return
-        }
-
-        if keyPath == #keyPath(AVPlayerItem.status) {
-            let status: AVPlayerItem.Status
-            if let statusNumber = change?[.newKey] as? NSNumber {
-                status = AVPlayerItem.Status(rawValue: statusNumber.intValue)!
-            } else {
-                status = .unknown
-            }
-
-            // Switch over status value
-            if status == .readyToPlay {
-                playerAnalytics?.ready { result in
-                    switch result {
-                    case .success:
-                        print("video is ready to be played")
-                    case let .failure(error):
-                        print("video is not ready to be played: \(error)")
-                    }
+    
+    override public func observeValue(
+        forKeyPath keyPath: String?,
+        of _: Any?,
+        change: [NSKeyValueChangeKey: Any]?,
+        context _: UnsafeMutableRawPointer?
+    ) {
+        if keyPath == "status" {
+            playerAnalytics?.ready { result in
```

Comment:
As ready is now call each time status changed, you should keep:
```
 if status == .readyToPlay
```

Reply:
It was not called on live, the way it was done before